ports/mimxrt/boards/MIMXRT1010_EVK/mpconfigboard.mk

```diff
@@ -8,6 +8,7 @@ MICROPY_HW_FLASH_SIZE ?= 0x1000000  # 16MB
 
 JLINK_PATH ?= /media/RT1010-EVK/
 JLINK_COMMANDER_SCRIPT = $(BUILD)/script.jlink
+BOARD_QTMR_SUPPORT = 0
```

Comment:
Is this an existing config value or a new one?  If new, can it be something like `MICROPY_HW_QTMR_SUPPORT`, to match others?

Reply:
OK. I changed it. It has a kind of double negation logic, such that it only has to be defined for the MIMXRT1011 devices.